src/tasks/entities/task.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+
+@Entity({ name: 'tasks' })
```

Comment:
```suggestion
@Entity('tasks')
```
Thế này là cũng được rồi em